gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -123,22 +130,23 @@ def print_help(self):
             if self.source != ""
             else "\nSource: ?\n"
         )
+        help_text += self.price_str
```

Comment:
```
Coin: bitcoin
Source: CoinGecko
Current Price: 46128.83 usd
Performance in interval (1day): 0.86%
Performance since specified start date (2021-01-04 00:00:00): 46.37%
```

could we have instead: 
```
Performance since 2021-01-04: 46.37%
```
without the time.

And also capitalize USD